### PR TITLE
Fix Dokku Destoy on PR Close

### PR DESCRIPTION
**Pull request by kelvinkipruto**
## Description

This PR fixes a bug where Dokku did not destroy a preview app on PR close

## Type of change

- [x] Bug Fix

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by VinneyJ — APPROVED** [review]

File: .github/workflows/codeforafrica-deploy-review-app.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ on:
       - "apps/codeforafrica/**"
       - "Dockerfile.codeforafrica"
       - ".github/workflows/codeforafrica-deploy-review.yml"
+    types: [opened, reopened, edited, closed, synchronize]
```
Comment (kilemensi) [inline .github/workflows/codeforafrica-deploy-review-app.yml:9]: For the slow ones such as myself, what does this add? Or put another way, what are the default `types`?

Comment (kelvinkipruto) [reply .github/workflows/codeforafrica-deploy-review-app.yml:9]: @kilemensi From the [docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request)

> By default, a workflow only runs when a pull_request event's activity type is opened, synchronize, or reopened. To trigger workflows by different activity types, use the types keyword.

File: .github/workflows/codeforafrica-deploy-review-app.yml [hunk 2]
```diff
@@ -6,6 +6,7 @@ on:
       - "apps/codeforafrica/**"
       - "Dockerfile.codeforafrica"
       - ".github/workflows/codeforafrica-deploy-review.yml"
```
Comment (koechkevin) [inline .github/workflows/codeforafrica-deploy-review-app.yml:8]:
```suggestion
      - ".github/workflows/codeforafrica-deploy-review-app.yml"
```

File: .github/workflows/codeforafrica-deploy-review-app.yml (hunk 1 above)
Comment (kilemensi) [reply .github/workflows/codeforafrica-deploy-review-app.yml:9]: Nice... can we add this as a comment above the types line? i.e. We need `closed` `edited` in addition to the default events; `opened`, `synchronize`, ad `reopened`.

What does edited do?

Comment (kelvinkipruto) [reply .github/workflows/codeforafrica-deploy-review-app.yml]: Good catch @kilemensi. Edited runs when a pull request's description or labels are edited. I've removed it.

**Comment by koechkevin** [comment]
@kelvinkipruto we might then need to update the condition for on merge a pr since we have only set for when it is closed.

Check line 92

File: .github/workflows/codeforafrica-deploy-review-app.yml [hunk 3]
```diff
@@ -5,7 +5,9 @@ on:
     paths:
       - "apps/codeforafrica/**"
       - "Dockerfile.codeforafrica"
-      - ".github/workflows/codeforafrica-deploy-review.yml"
+      - ".github/workflows/codeforafrica-deploy-review-app.yml"
+      # By default, a workflow only runs when a pull_request event's activity type is opened, synchronize, or reopened. We need to add closed to the list of activity types to ensure that the workflow runs when a pull request is closed.
```
Comment (kilemensi) [inline .github/workflows/codeforafrica-deploy-review-app.yml:9]: Why such a long line? Limit to 80 - 100 chars (whatever makes sense).

**Review by kilemensi — APPROVED** [review]

**Comment by kelvinkipruto** [comment]
@koechkevin Good catch Let me update it.

**Comment by kilemensi** [comment]
> we might then need to update the condition for on merge a pr since we have only set for when it is closed
>

Again, for the rest of us still learning the ropes, when a PR is merged, isn't it also [`closed`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-pull_request-workflow-when-a-pull-request-merges) @koechkevin / @kelvinkipruto ?

**Review by koechkevin — APPROVED** [review]

**Pull request merged**